### PR TITLE
Additional images given relevant alt attributes

### PR DESCRIPTION
**Pull request by leeenglestone**
I have provided missing and unique alt attributes for other images on the site.

For issue #130

### CODE REVIEW

File: src/Pages/About.cshtml [hunk 1]
```diff
@@ -375,27 +375,27 @@
 				<h2 id="board-of-directors">Board of Directors</h2>
 				<div class="row">
 					<div class="col-md-3">
-						<img height="120" class="rounded-image" alt="ProfilePicture" src="https://avatars3.githubusercontent.com/u/19977?s=400&v=4">
+						<img height="120" class="rounded-image" alt="Phil Haak" src="https://avatars3.githubusercontent.com/u/19977?s=400&v=4">
```
Comment (haacked) [inline src/Pages/About.cshtml:378]: My last name is spelled "Haack" :smile:

File: src/Pages/About.cshtml [hunk 2]
```diff
@@ -375,27 +375,27 @@
...
 						<h4 class="col-md-3-name"><a href="https://election.dotnetfoundation.org/campaign-2019/phil-haack.html">Phil Haack</a></h4>
 					</div>
 					<div class="col-md-3">
-						<img height="120" class="rounded-image" alt="ProfilePicture" src="https://avatars2.githubusercontent.com/u/1142958?s=400&u=35ad462d9638ab4560868ec9c12087b7c0039c75&v=4">
+						<img height="120" class="rounded-image" alt="Ben Adams" src="https://avatars2.githubusercontent.com/u/1142958?s=400&u=35ad462d9638ab4560868ec9c12087b7c0039c75&v=4">
 						<h4 class="col-md-3-name"><a href="https://election.dotnetfoundation.org/campaign-2019/ben-adams.html">Ben Adams</a></h4>
 					</div>
 					<div class="col-md-3">
-						<img height="120" class="rounded-image" alt="ProfilePicture" src="https://avatars0.githubusercontent.com/u/110101?s=400&v=4">
+						<img height="120" class="rounded-image" alt="Sara Chips" src="https://avatars0.githubusercontent.com/u/110101?s=400&v=4">
```
Comment (haacked) [inline src/Pages/About.cshtml:386]: Chipps has two "p"s

File: src/Pages/About.cshtml (hunk 1 above)
Comment (leeenglestone) [reply src/Pages/About.cshtml:378]: ok,  will fix later. ta

**Comment by clairernovotny** [comment]
Closing as this needs to be done in the new site https://github.com/dotnet-foundation/website

**Pull request closed without merging**